### PR TITLE
feat: add Ice Floe collab and update RIDDC

### PR DESCRIPTION
**Pull request by tdivoll**
I added the Ice Floe collaboration, added myself to the RIDDC collab, and added a tag for `climate-science`

### CODE REVIEW

File: our-work/collaborations/wilhelmus.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+title: Wilhelmus Lab
+id: wilhelmus-lab
+description: The Wilhelmus Lab takes an engineering approach to solve scientific problems related to fluid mechanics and transport mechanisms. One example is tracking ice floes in satellite imagery to understand velocity and turblence in the Arctic Ocean. CCV collaborates on this project with Dr. Wilhelmus to translate the existing code base into a module that can be parameterized and used by researchers. This includes looking for processing efficiencies in the process.
+tags:
+  - active
+  - computer-vision
+  - climate-science
```
Comment (mirestrepo) [inline our-work/collaborations/wilhelmus.yml:7]:
```suggestion
  - climate-science
  - engineering
```

File: schemas/collaborations.yml [hunk 2]
```diff
@@ -25,7 +25,8 @@ id: "enum(
 	  'mcdonald-lab',
 	  'mchugh-lab',
 	  'rothman-lab',
-	  'spade-lab'
+	  'spade-lab',
+    'wilhelmus-lab'
```
Comment (mirestrepo) [inline schemas/collaborations.yml:29]: spacing off?

**Review by mirestrepo — APPROVED** [review]
This looks great. I left few suggestions

File: schemas/collaborations.yml (hunk 2 above)
Comment (hollandjg) [reply schemas/collaborations.yml]: Should be fixed now, I merged "main" where it was repaired.

File: our-work/collaborations/wilhelmus.yml (hunk 1 above)
Comment (tdivoll) [reply our-work/collaborations/wilhelmus.yml:7]: added!

File: schemas/collaborations.yml (hunk 2 above)
Comment (tdivoll) [reply schemas/collaborations.yml]: confirmed

**Pull request merged**